go/internal/initutil/manager_test.go

```diff
@@ -146,10 +186,9 @@ func TestTwoConcurrentManagers(t *testing.T) {
 }
 
 func TestCloseByContext(t *testing.T) {
-	// FIXME: defer goleak.VerifyNone(t, goleak.IgnoreCurrent())
-
+	defer verifyRunningLeakDetection(t)
 	ctx, cancel := context.WithCancel(context.Background())
-	defer cancel()
```

Comment:
I don't get why you move this defer call at the end without defer?

If I'm not wrong, this defer will be stacked and be called "before" the "verifyRunning" one, so for me, it's the same behavior, except that cancel will be called even if a "require.something" make the call return earlier

Reply:
Yes, at here I did not sure the order of deferred functions are called.
So that why move cancel() to end without defer to ensure cancel() is called before verifyRunningLeakDetection() is called.

Reply:
Reverted it.